packages/manager/src/features/NodeBalancers/NodeBalancerDetail/NodeBalancerConfigurations.tsx

```diff
@@ -1164,6 +1090,69 @@ class NodeBalancerConfigurations extends React.Component<
     const clampedValue = clampNumericString(0, Number.MAX_SAFE_INTEGER)(value);
     this.updateState(lens, L, callback)(clampedValue);
   };
+
+  render() {
+    const { nodeBalancerLabel } = this.props;
+    const {
+      configErrors,
+      configSubmitting,
+      configs,
+      hasUnsavedConfig,
+      panelMessages,
+    } = this.state;
+
+    const isNodeBalancerReadOnly = this.isNodeBalancerReadOnly();
+
+    return (
+      <div>
+        <DocumentTitleSegment
+          segment={`${nodeBalancerLabel} - Configurations`}
+        />
+        {Array.isArray(configs) &&
+          configs.map(
+            this.renderConfig(panelMessages, configErrors, configSubmitting)
+          )}
+
+        {!hasUnsavedConfig && (
+          <Box sx={{ marginTop: '16px' }}>
+            <StyledConfigsButton
+              buttonType="outlined"
+              data-qa-add-config
+              disabled={isNodeBalancerReadOnly}
+              onClick={() => this.addNodeBalancerConfig()}
+            >
+              {configs.length === 0
+                ? 'Add a Configuration'
+                : 'Add Another Configuration'}
+            </StyledConfigsButton>
+          </Box>
+        )}
+
+        <ConfirmationDialog
+          actions={
+            <NodeBalancerConfigConfirmationActions
+              isLoading={this.state.deleteConfigConfirmDialog.submitting}
+              onClose={this.onCloseConfirmation}
+              onDelete={this.deleteConfig}
+            />
```

Comment:
Let's just use the `ActionsPanel` directly and delete the `NodeBalancerConfigConfirmationActions` component
```suggestion
                        <ActionsPanel
              primaryButtonProps={{
                'data-testid': 'confirm-cancel',
                label: 'Delete',
                loading: this.state.deleteConfigConfirmDialog.submitting,
                onClick: this.deleteConfig,
              }}
              secondaryButtonProps={{
                'data-testid': 'cancel-cancel',
                label: 'Cancel',
                onClick: this.onCloseConfirmation,
              }}
              style={{ padding: 0 }}
            />
```